src/main/java/frc/trigon/robot/subsystems/collector/Collector.java

```diff
@@ -0,0 +1,45 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Collector extends SubsystemBase {
+    private static final WPI_TalonFX
+            openingMotor = CollectorConstants.OPENER_MOTOR,
+            collectingMotor = CollectorConstants.COLLECTOR_MOTOR;
+
+    private final static Collector INSTANCE = new Collector();
+
+    private Collector() {
+
+    }
+
+    public static Collector getInstance() {
+        return INSTANCE;
+    }
+
+    private void collect() {
+        openingMotor.set(CollectorConstants.OPENING_POWER);
```

Comment:
You should probably use `ControlMode.Position`. Ill discuss with @laviRZ and get back to you

Reply:
We always put on power, with a current limit.

Reply:
you sure? I guess that's ok... pid is unnecessary here

Reply:
Where did I say PID? We constantly give power, with a simple current limit...

Reply:
You didn't say PID... you misunderstood my comment